docs/configs/html/conf.py

```diff
@@ -177,6 +177,10 @@
     'ledger-api-test-tool-maven': ('https://repo1.maven.org/maven2/com/daml/ledger-api-test-tool/{}/%s-{}.jar'.format(version, version), None)
 }
 
+rst_prolog = """
```

Comment:
Is this necessary? Can't you just use the `|release|` substitution in RST?
Ie doesn't 
```
`https://github.com/digital-asset/daml/releases/download/v|release|/daml-sdk-|release|-windows.exe`__
```
do the trick?

Reply:
I did not manage to get the substitution to work inside a link, though it does indeed work well in the text. Happy to switch if you manage to make it work.

Reply:
I spent quite some time trying to get this to work and gave up in the end. What does however work is the `extlinks` plugin which I added back then. See line 173 which has almost what you need here except for the file ending. One option would be to just move the file ending to the call site or alternatively duplicate `github-asset` and come up with a somewhat sensible name.

Reply:
I'm not sure how you would "move the file ending to the call site". We want the file ending to be part of the URL, and we can't have the release string at the call site, and we only get one argument for the URL.

This parses the entire quoted string as the artifact name within the URL:

```
:gihtub-asset:`link text <asset-name>.zip`
```

This does not substitute `{}`:

```
:gihtub-asset:`link text <asset-name-{}.zip>`
```

This does not substitute `|release|`:

```
:gihtub-asset:`link text <asset-name-|release|.zip>`
```

I was not very happy about the time it took me to realize `rst_prolog` existed and was the only way to achieve this, but now that I have, I must say I'm not quite sure what is wrong with it.

Reply:
I thought it might be possible to have two parameters. iirc the API of `extlinks` was annoyingly limited so I wouldn’t be too surprised if it isn’t possible. We could still have `github-asset-zip` and `github-asset-exe`.

As for `rst_prolog`. I don’t care too much if we use `extlinks` or `rst_prolog` but I’d like this to be consistent. Using one for the Windows installer and the other for protobuf tarballs is pretty confusing.

Reply:
Moved all three to `rst_prolog` then, as it makes more sense for the uses we have so far: `extlinks` requires exactly one parameter, and in none of the use-cases we have do we actually need one.